### DISCUSSION TITLE
Self-hosting Backend and Frontend Independently

### DISCUSSION THREAD

**Discussion by bryan-gc** [Unleash/unleash#8436] [Q&A]
I was wondering if there is a way to separate the backend and frontend of Unleash. I noticed that there is an old [repo](https://github.com/Unleash/unleash-frontend) for the admin UI, but it's archived (I think it's the same code from this [folder](https://github.com/Unleash/unleash/tree/main/frontend)?). At my work, we would like to self-host Unleash and create different instances with just the backend. Is this possible nowadays?

**Comment by chriswk** [comment]
Hi @bryan-gc
The frontend is automatically hosted in the backend server. You can still deploy the rolledup frontend folder separately. We host our own frontend on cloudfront CDN. You'd need to find a way to inject the backend URL into the frontend at build time; and configure CORS correctly on the backend. I'll ping @FredrikOseberg

**Comment by FredrikOseberg** [comment]
@bryan-gc 

You can provide Unleash with a cdnPrefix option or environment variable CDN_PREFIX that points to a location (URL) that contains a built frontend.
